src/components/calculator.js

```diff
@@ -1,35 +1,46 @@
 import React from 'react';
 import '../calculator.css';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      total: 0,
```

Comment:
- [ ] For this milestone, we are expecting you to practice what you have learned until now; like passing props to child components, using lifecycle methods, lifting state up. You should use these concepts in your app.

Tip:
_You should divide your `Calculator` component into child components in order to practice these concepts. (For example, you can create a display component for displaying calculations and a keypad component for the keys.)_



From the project requirements ⬇️ 
>Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift the state up.

Reply:
@sinansevgi Thank you for your Feedback but can you give further explanation, please.